gluon/scheduler.py

```diff
@@ -41,7 +41,7 @@
 from gluon.scheduler import Scheduler
 
 def demo1(*args,**vars):
-    print 'you passed args=%s and vars=%s' % (args, vars)
+    print('you passed args=%s and vars=%s') % (args, vars)
```

Comment:
I find this cleaner : 
print('you passed args=%s and vars=%s' % (args, vars))

Or even : 
print('you passed args={} and vars={}'.format(args, vars))